profiles/rawls-service-perimeters/terraform/rawls-service-perimeters/perimeters.tf

```diff
@@ -45,4 +45,7 @@ resource "google_access_context_manager_service_perimeter" "service-perimeter" {
       status[0].resources
     ]
   }
+  provisioner "local-exec" {
```

Comment:
It'd probably be good to comment here about what this does and doesn't do. thinking about this some more yesterday, I think this will error if terraform re-runs the script, as the resource will already exist in SAM. That might cause issues.

Reply:
I believe the exit code for the curl command when it gets a 409 back will still be 0, so Terraform should be okay if it re-runs the script. That said, we may want to actually check if we get a 409 or a 2xx back and log that result.

Agreed on a comment here and in the script about what it does and doesn't do with respect to perimeter setup stuff, especially since it isn't updating existing perimeters like one might expect from Terraform